ui/packages/consul-ui/tests/steps/interactions/form.js

```diff
@@ -57,21 +57,23 @@ export default function(scenario, find, fillIn, triggerKeyEvent, currentPage) {
         } catch (e) {
           obj = currentPage();
         }
-        const res = Object.keys(data).reduce(function(prev, item, i, arr) {
+        const res = Object.keys(data).reduce(async function(prev, item, i, arr) {
+          await prev;
+
           const name = `${obj.prefix || property}[${item}]`;
           if (negative) {
             try {
-              fillInElement(prev, name, data[item]);
+              await fillInElement(obj, name, data[item]);
               throw new TypeError(`${item} is editable`);
             } catch (e) {
               if (e instanceof TypeError) {
                 throw e;
               }
             }
           } else {
-            return fillInElement(prev, name, data[item]);
+            return await fillInElement(obj, name, data[item]);
           }
-        }, obj);
+        }, Promise.resolve());
         await new Promise(resolve => setTimeout(resolve, 0));
```

Comment:
Guessing we can remove the 'nextTickPromise' on line 77  here now we've upgraded ember-test-helpers? Also, do we need to `return await` instead of just `return` on line 74?

Reply:
We spoke about this offline and @LevelbossMike decided the extra await here was necessary, so skipping this